packages/terra-clinical-result/src/ClinicalResult.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import classNames from 'classnames/bind';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconUnverified from 'terra-icon/lib/icon/IconDiamond';
+import ResultError from './common/other/_ResultError';
+import NoData from './common/other/_KnownNoData';
+import Observation from './common/observation/_Observation';
+import observationPropShape from './proptypes/observationPropTypes';
+import { checkIsStatusInError, ConditionalWrapper } from './common/utils';
+import styles from './ClinicalResult.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Result Object with the clinical result data.
+   */
+  resultData: observationPropShape,
+  /**
+   * Visually hides the unit of measure when presented in a series of side-by-side columns of the same unit.
+   */
+  hideUnit: PropTypes.bool,
+  /**
+   * Whether or not the text should be truncated in display. Restricts clinical result details each to one line.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   * Override that shows an Error display. Used when there is a known error or problem when retrieving or assembling the clinical result data.
+   */
+  hasResultError: PropTypes.bool,
+  /**
+   * Override that shows a known "No Data" display. Used when there is known to be no value for a given clinical result concept at a specific datetime.
+   */
+  hasResultNoData: PropTypes.bool,
+  /**
+   * @private
+   * The intl object to be injected for translations.
+   */
+  intl: intlShape.isRequired,
+};
+
+const defaultProps = {
+  resultData: {},
+  hideUnit: false,
+  isTruncated: false,
+  hasResultError: false,
+  hasResultNoData: false,
+};
+
+const createIcons = (resultData) => (resultData.isUnverified
+  ? (<IconUnverified className={cx('icon-unverified')} />)
+  : (
+    <React.Fragment>
+      {resultData.isModified ? (<IconModified className={cx('icon-modified')} />) : null}
+      {resultData.hasComment ? (<IconComment className={cx('icon-comment')} />) : null}
+    </React.Fragment>
+  )
+);
+
+const createSecondaryDisplays = (resultData) => (
+  <React.Fragment>
+    {resultData.conceptDisplay ? (<div className={cx('concept-display')}>{resultData.conceptDisplay}</div>) : null}
+    {resultData.datetimeDisplay ? (<div className={cx('datetime-display')}>{resultData.datetimeDisplay}</div>) : null}
+  </React.Fragment>
+);
+
+const createClinicalResultDisplay = (resultData, hideUnit, isTruncated, hasResultError, hasResultNoData) => {
+  if (hasResultError) {
+    return <ResultError />;
+  }
+  if (hasResultNoData) {
+    return <NoData />;
+  }
+  const isStatusInError = checkIsStatusInError(resultData);
+  const decoratedResultClassnames = cx([
+    'decorated-result-display',
+    { truncated: isTruncated },
+    { 'status-in-error': isStatusInError },
```

Comment:
Should this have returned the ResultError view?

Reply:
slightly confusing, but there is a difference between an "Error" and when something has been documented and then uncharted, which then has a status of "entered-in-errror" (goes with other [status proptypes](https://github.com/cerner/terra-clinical/blob/db68b0a747a778e3fa04fba96a786bd9cb85f1f2/packages/terra-clinical-result/src/proptypes/statusPropTypes.js) like "final", "corrected", "cancelled" etc.)

- ResultError = either known error from `hasResultError` prop or fallback for when result data is malformed
- Status-In-Error = strike-thru for normal result component or alternate template in flowsheet cell